exporter/elasticsearchexporter/config.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package elasticsearchexporter
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/config/configtls"
+)
+
+// Config defines configuration for Elastic exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"`
+
+	// Endpoints holds the Elasticsearch URLs the exporter should send events to.
+	//
+	// This setting is required if CloudID is not set and if the
+	// ELASTICSEARCH_URL environment variable is not set.
+	Endpoints []string `mapstructure:"endpoints"`
+
+	// CloudID holds the cloud ID to identify the Elastic Cloud cluster to send events to.
+	// https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html
+	//
+	// This setting is required if no URL is configured.
+	CloudID string `mapstructure:"cloudid"`
+
+	// Workers configures the number of workers publishing bulk requests.
+	Workers int `mapstructure:"workers"`
```

Comment:
Consider "num_workers" to be consistent with NumConsumers in our sending_queue helper.

Reply:
+1
Done